graspologic/embed/ase.py

```diff
@@ -67,7 +67,7 @@ class AdjacencySpectralEmbed(BaseSpectralEmbed):
     Attributes
     ----------
     n_features_in_: int
-        Number of features passed to the 
+        Number of features passed to the
         :func:`~graspologic.embed.AdjacencySpectralEmbed.fit` method.
     latent_left_ : array, shape (n_samples, n_components)
         Estimated left latent positions of the graph.
```

Comment:
This commit made me realize it'd probably be smart to make a github action to just run black automatically every time a commit gets pushed. Not too familiar with github actions so I'm not sure how easy that'd be